### DISCUSSION TITLE
[Script request] Alby Hub

### DISCUSSION THREAD

**Discussion by holocronology** [community-scripts/ProxmoxVE#629] [Request script]
### Application Name

Alby Hub

### Website

https://getalby.com/nwc/server

### Description

Alby Hub is a new and powerful self-custodial bitcoin lightning wallet ([learn more](https://guides.getalby.com/user-guide/v/alby-account-and-browser-extension/alby-hub/introduction)) that can run in a cloud or on your own device.

This might hold some keys: https://github.com/getAlby/hub/tree/master/scripts/linux-x86_64

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by clartek** [comment]
This is easy enough to do yourself. Just use the Ubuntu LXC script and run the script Alby gives you to install. I've been using my node this way for months, and it works great.
